app/controllers/observations_controller/form_helpers.rb

```diff
@@ -16,20 +16,38 @@
 module ObservationsController::FormHelpers
   private
 
+  # Note this array syntax: The nested attributes MUST come last.
+  # VSCode won't allow nested square brackets (?).
   def permitted_observation_args
     [:place_name, :where, :lat, :long, :alt, :when, "when(1i)", "when(2i)",
-     "when(3i)", :notes, :specimen, :thumb_image_id, :is_collection_location,
-     :gps_hidden]
+     "when(3i)", :specimen, :thumb_image_id, :is_collection_location,
+     :gps_hidden, { notes: [:Other] }]
   end
 
   def update_permitted_observation_attributes
     @observation.attributes = permitted_observation_params || {}
   end
 
+  # NOTE: You MUST call `to_h` on the permitted params, because param nesting.
+  # As of rails 5, params are an ActionController::Parameters object,
+  # not a hash.
   def permitted_observation_params
     return unless params[:observation]
 
-    params[:observation].permit(permitted_observation_args)
+    params[:observation].permit(permitted_observation_args).to_h
+  end
+
+  # Symbolize keys; delete key/value pair if value blank
+  # Also avoids whitelisting issues
+  def notes_to_sym_and_compact
+    return Observation.no_notes unless notes_param_present?
+
+    symbolized = params[:observation][:notes].to_unsafe_h.symbolize_keys
+    symbolized.compact_blank!
+  end
+
+  def notes_param_present?
+    params[:observation] && params[:observation][:notes].present?
```

Comment:
May I ask a favor?  If it's really important to you to move methods around like this, please ignore the request, but...  Where possible, it is better from the perspective of reviewers if you don't move methods around.  When you do there is no way to (easily) see what (if anything) has been changed.  It just makes reviewing and understanding your PR more difficult.  I'm sure there will be excellent cases for moving things, but just in case you aren't aware of that side-effect (messing up the diffs), I thought I'd throw that out there.

Reply:
Sure thing. I'll move in a separate PR. 

Moved these either because they're used by both `edit_and_update` and `new_and_create` or were closely interrelated with methods that were.